content/docs/styled-system/color-mode.mdx

```diff
@@ -80,36 +90,75 @@ export default theme
-  modification/initial load of the colorMode this value will be used
+> **Note** The hook `useColorMode` let you update color mode of your app in every cases. [Learn more](#changing-color-mode)
+
+```tsx live=false
+// A.
+// System sets initial value.
+// App subscribes to system color mode changes.
+const config: ThemeConfig = {
+  initialColorMode: 'system',
+  useSystemColorMode: true,
+}
+
+// B.
+// System sets initial value.
+// App color mode is detached from system color mode changes.
+const config: ThemeConfig = {
+  initialColorMode: 'system',
+  useSystemColorMode: false,
+}
+
+// C.
+// You choose initial value.
+// App color mode is detached from system color mode changes.
+const config: ThemeConfig = {
+  initialColorMode: 'dark', // 'dark' | 'light'
+  useSystemColorMode: false,
+}
+
+
+// D.
+// You choose initial value.
+// App subscribes to system color mode changes.
+const config: ThemeConfig = {
+  initialColorMode: 'dark', // 'dark' | 'light'
+  useSystemColorMode: true,
+}
+```
+
+#### Behavior of ColorMode
+
+The current hierarchy of how chakra resolves the color mode:
 
-- if `colorModeManager` = `localStorage` and a value is defined for
-  `chakra-ui-color-mode` this will be used
+- Get the color mode value in the specified storage (localStorage or cookie manager or custom)
 
-- if `initialColorMode` = `system` system-color will be used as default -
-  `initialColorMode` is the fallback if system color mode isn't resolved
+- If value doesn't exist, use the `initialColorMode` value specified.
+  - If the initial value is `system`, then we'll compute the color mode using
+    `matchMedia`
+  - Else, we use the initial value as is.
 
-- if `initialColorMode` = `'light'|'dark'` the corresponding value will be used
+- If user specifies `useSystemColorMode: true`, then we'll subscribe to color
+  mode changes from the operating system. It is no longer used to determine the
+  initial color mode. To achieve that, please use `initialColorMode: "system"`
 
-We currently accept 3 different values for `initialColorMode`:
-`light`,`dark`,`system`
+// TODO : Keep this ??? Is still valid ??
+- if `data-theme` prop is defined through e.g. `ColorModeScript` / after
+  modification/initial load of the colorMode this value will be used
 
 #### Difference between `initialColorMode='system'` and `useSystemColorMode`
 
-if `useSystemColorMode=true` we will always try to match the users
-`system`-color and fallback to `initialColorMode`. With this behavior, the
-colorMode toggle won't have any effect.
+`initialColorMode` value is used to determine the value on first visit, or after a storage reset.
 
-if `initialColorMode='system'` we will as well always try to match the users
-`system`-color and fallback to `light`. After the user has toggled the value,
-this value will be used.
+`useSystemColorMode` is a boolean that indicates if chakra must subscribes (and update) to operative system's color mode changes.
```

Comment:
Can we change this to: 
```suggestion
`useSystemColorMode` is a boolean that indicates if Chakra UI subscribes (and updates) based on the operating system's color mode changes.
```

Reply:
Included in commit 1d1ec8bb6a4a658357504dbde6b8077c8ac6cfb0